### PR TITLE
fix: Update readme

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

**Comment by gaocegege** [comment]
Preview: https://github.com/gaocegege/envd/tree/refine

/assign @kemingy @VoVAllen

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ envd (`ɪnˈvdɪ`) is a **machine learning development environment** for data sc
 
 🔁 **Repeatable builds & reproducible results** - You can reproduce the same dev environment on your laptop, public cloud VMs, or Docker containers, without any change in setup.
 
+<p align="center">
+  <img src="docs/images/demo.gif" width="600"/>
```
Comment (kemingy) [inline README.md:31]: This gif is huge. Can we upload it to discussion or issue or gist?

Comment (gaocegege) [reply README.md:31]: SGTM.

Comment (zwpaper) [reply README.md:31]: /hold

Comment (gaocegege) [reply README.md]: /hold-cancel

Comment (kemingy) [reply README.md]: Can delete the gif and squash

File: README.md [hunk 2]
```diff
@@ -105,11 +109,11 @@ We’re focused on helping data scientists and teams that develop AI/ML models.
 
 [**Set up a time to chat!**](https://forms.gle/9HDBHX5Y3fzuDCDAA)
 
----
+<!-- ---
 
 Before envd             |  After envd
 :-------------------------:|:-------------------------:
-![](./docs/images/before.svg)  |  ![](./docs/images/after.svg)
+![](./docs/images/before.svg)  |  ![](./docs/images/after.svg) -->
```
Comment (VoVAllen) [inline README.md:116]: This part is not commented out after rendering. Is this expected?

Comment (gaocegege) [reply README.md:116]: Yep, I think it is unnecessary, WDYT

Comment (VoVAllen) [reply README.md:116]: Looks good to me. Github markdown preview confused me

**Review by VoVAllen — APPROVED** [review]

**Pull request merged**